test/TestAsset/TestCase.php

```diff
@@ -31,19 +31,55 @@
 
 namespace OpusTest\TestAsset;
 
+use Opus\Config;
 use Opus\Log;
 use Opus\Log\LogService;
 use PHPUnit\Framework\TestCase as PHPUnitFrameworkTestCase;
+use Zend_Config;
 
+use function call_user_func;
 use function dirname;
+use function is_callable;
 
 class TestCase extends PHPUnitFrameworkTestCase
 {
+    private $baseConfig;
+
     public function setUp()
     {
         parent::setUp();
 
         Log::drop(); // just in case there is still an old logger cached
         LogService::getInstance()->setPath(dirname(__FILE__, 3) . '/build/log');
     }
+
+    /**
+     * Overwrites selected properties of current configuration.
+     *
+     * @note A test doesn't need to backup and recover replaced configuration as
+     *       this is done in setup and tear-down phases.
+     * @param array         $overlay properties to overwrite existing values in configuration
+     * @param null|callable $callback callback to invoke with adjusted configuration before enabling e.g. to delete some options
+     * @return Zend_Config reference on updated configuration
+     */
+    protected function adjustConfiguration($overlay, $callback = null)
```

Comment:
Die Funktion ist nicht notwendig, da keine Konfiguration automatische geladen wird. Jeder Test kann einfach einen eigene Konfiguration setzen.

Reply:
Ich verstehe warum das hier auf den ersten Blick sinn macht. Diese Funktion ist jetzt aber für alle Tests verfügbar. Es wird aber nicht für alle Tests in `setUp` eine Konfiguration angelegt, die dann modifiziert werden kann. Wenn die Konfiguraton aber leer ist, dann muss sie auch nicht mit "adjust" modifiziert werden. Bitte innerhalb von `FileTypesTest` lösen.

Reply:
Am besten indem die Konfiguration in jedem Test nach Bedarf gesetzt wird. Dann ist auch besser zu erkennen welche Einträge überhaupt relevant sind.

Reply:
Meinetwegen. Ich finde nur Dateien editieren einfacher und nachvollziehbarer als Dateien verschieben.

Reply:
Dann zieh bitte auch gleich den Branch **commonNamespace** nach.